src/interfaces/augment-api-tx.ts

```diff
@@ -762,7 +762,7 @@ declare module '@polkadot/api/types/submittable' {
        * the `origin` falls below minimum bond and is removed lazliy in `withdraw_unbonded`.
        * # </weight>
        **/
-      bond: AugmentedSubmittable<(controller: AccountId | string | Uint8Array, value: Compact<BalanceOf> | AnyNumber | Uint8Array, payee: RewardDestination | { Staked: any } | { Stash: any } | { Controller: any } | { Account: any } | string | Uint8Array) => SubmittableExtrinsic<ApiType>>;
+      bond: AugmentedSubmittable<(controller: AccountId | string | Uint8Array, value: Compact<BalanceOf> | AnyNumber | Uint8Array, payee: RewardDestination | { Stash: any } | { Controller: any } | { Account: any } | string | Uint8Array) => SubmittableExtrinsic<ApiType>>;
```

Comment:
how do we ensure this won't be overwritten when regenerating?

Reply:
I haven't changed this manually.. This is auto generated by yarn build after adding RewardDestination in definition.ts for staking..

Reply:
perfect!